lib/luhn_cases.rb

```diff
@@ -0,0 +1,27 @@
+require 'exercise_cases'
+
+class LuhnCase < OpenStruct
+  def name
+    'test_%s' % description.tr('- ', '__')
+  end
+
+  def work_load
```

Comment:
No, just the README.
I've made a PR to rename the old occurrences to try and clear this up: https://github.com/exercism/xruby/pull/561